packages/studio-ui/src/store/useStudioStore.ts

```diff
@@ -49,6 +50,7 @@ const useStudioStore = create<StudioStore>()(
         ),
         studioConfig: lens(createStudioConfigSlice),
         studioEnvData: lens(createStudioEnvDataSlice),
+        studioGitData: lens(createStudioGitDataSlice),
```

Comment:
I think just gitData is fine

Reply:
arguably studioEnvData can just be envData too but whatever

Reply:
renamed both!